lib/payrix/configuration.rb

```diff
@@ -3,20 +3,43 @@
 module Payrix
   # Use this class to configure API parameters such as API URL, API key, etc.
   class Configuration
-    attr_accessor :api_key, :session_key, :test_mode
+    attr_accessor :api_key, :session_key
+    attr_reader :environment
 
     def initialize
       @api_key = ''
       @session_key = ''
-      @test_mode = false
+      @environment = Payrix::ENVIRONMENTS.fetch(:sandbox)
```

Comment:
Change the default environment setting to sandbox. This is done to foster more safety when using the library, and prevent making accidental mutations in the production environment.